tests on py 3.x fail with a pytest-pep8 related warning/failure

See e.g. https://travis-ci.org/github/MDAnalysis/pytng/jobs/719409548 

```
==================================== ERRORS ====================================

________________________ ERROR collecting test session _________________________

../../../virtualenv/python3.6.7/lib/python3.6/site-packages/pluggy/hooks.py:289: in __call__

    return self._hookexec(self, self.get_hookimpls(), kwargs)

../../../virtualenv/python3.6.7/lib/python3.6/site-packages/pluggy/manager.py:87: in _hookexec

    return self._inner_hookexec(hook, methods, kwargs)

../../../virtualenv/python3.6.7/lib/python3.6/site-packages/pluggy/manager.py:81: in <lambda>

    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,

../../../virtualenv/python3.6.7/lib/python3.6/site-packages/pytest_pep8.py:38: in pytest_collect_file

    return Pep8Item(path, parent, pep8ignore, config._max_line_length)

../../../virtualenv/python3.6.7/lib/python3.6/site-packages/_pytest/nodes.py:95: in __call__

    warnings.warn(NODE_USE_FROM_PARENT.format(name=self.__name__), stacklevel=2)

E   pytest.PytestDeprecationWarning: Direct construction of Pep8Item has been deprecated, please use Pep8Item.from_parent.

E   See https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent for more details.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,13 +13,13 @@ branches:
 
 install:
   - pip install --upgrade pytest
-  - pip install codecov pytest-cov pytest-pep8
+  - pip install codecov pytest-cov
   - pip install --upgrade numpy cython
   - if [[ $TRAVIS_PYTHON_VERSION == "3.6" ]]; then pip install --upgrade sphinx sphinx-sitemap; fi
   - pip install -e .
 
 script:
-  - py.test -v --cov pytng --pep8 pytng tests
+  - pytest -v --cov pytng pytng tests
 
 after_success:
   - codecov

